test_scripts/Polices/Policies_Security/Trigger_PTU_NO_Certificate/049_ATF_P_TC_default_PTU_brings_certificate_RPC_ForceProtectedService_OFF.lua

```diff
@@ -0,0 +1,158 @@
+-- and PolicyTableUpdate is valid and brings "certificate" at "module_config" section
+-- and app sends StartService (<any_serviceType>, encypted=true) to SDL
+-- SDL must start TLSHandshake sequence (by sending TLS Hello to this mobile app)
+--
+-- 1. Used preconditions:
+-- RPC SetAudioStreamingIndicator is allowed by policy
+-- ForceProtectedService is set to OFF in .ini file
+-- Default app exists in LP, no certificate in module_config
+-- Register and activate application.
+-- Send StartService(serviceType = 7 (RPC))
+-- -> SDL should trigger PTU: SDL.OnStatusUpdate(UPDATE_NEEDED)
+-- -> SDL should not respond to StartService_request
+-- -> SDL should not process request to HMI
+--
+-- 2. Performed steps
+-- Send correct policy file, certificate exists in module_config
+--
+-- Expected result:
+-- 1. SDL sends SDL.OnStatusUpdate(UP_TO_DATE)
+-- 2. SDL process request to HMI
+-- 3. SDL should return StartServiceACK, encrypt = true to RPC
+---------------------------------------------------------------------------------------------
+
+--[[ General configuration parameters ]]
+config.deviceMAC = "12ca17b49af2289436f303e0166030a21e525d266e209267433801a8fd4071a0"
+config.application1.registerAppInterfaceParams.appHMIType = {"DEFAULT"}
+config.application1.registerAppInterfaceParams.isMediaApplication = false
+
+--[[ Required Shared libraries ]]
+local commonFunctions = require ('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local commonPreconditions = require('user_modules/shared_testcases/commonPreconditions')
+local commonTestCases = require('user_modules/shared_testcases/commonTestCases')
+local testCasesForPolicyCeritificates = require('user_modules/shared_testcases/testCasesForPolicyCeritificates')
+local events = require('events')
+local Event = events.Event
+
+--[[ General Precondition before ATF start ]]
+commonFunctions:write_parameter_to_smart_device_link_ini("ForceProtectedService", "Non")
+testCasesForPolicyCeritificates.update_preloaded_pt(config.application1.registerAppInterfaceParams.appID, false)
+testCasesForPolicyCeritificates.create_ptu_certificate_exist(true,false)
+commonSteps:DeletePolicyTable()
+commonSteps:DeleteLogsFiles()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('user_modules/AppTypes')
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+
+function Test:Precondition_ActivateApp()
+  commonSteps:ActivateAppInSpecificLevel(self, self.applications[config.application1.registerAppInterfaceParams.appName])
+  EXPECT_NOTIFICATION("OnHMIStatus", {systemContext = "MAIN", hmiLevel = "FULL"})
+end
+
+function Test:Precondition_First_StartService()
+  self.mobileSession.correlationId = self.mobileSession.correlationId + 1
+  local msg = {
+    serviceType = 7,
+    frameType = 0,
+    frameInfo = 1,
+    encryption = true,
+    rpcCorrelationId = self.mobileSession.correlationId
+  }
```

Comment:
@dboltovskyi, yes we expect only StartService for RPC, without SetAudioStreamingIndicator. Specification for SetAudioStreamingIndicator in description is removed.